data/sites.yml

```diff
@@ -340,6 +340,8 @@ built:
     title: "PRX.co: PR for startups and small businesses"
   - url: https://www.ecommwar.com/
     title: eCommWar - Top Open-Source E-Commerce Platforms
+  - url: https://www.lionit.co.uk/
```

Comment:
Looks like this cutting into the eCommWar object, instead of creating its own. Meaning, the `source` key for eCommWar on Line 345 is now tied to your website. 

Can you move your addition to the bottom of the list?

Reply:
@liononline Any update on this?